lib/gui/etcher.js

```diff
@@ -21,16 +21,28 @@ const path = require('path')
 const _ = require('lodash')
 const { autoUpdater } = require('electron-updater')
 const Bluebird = require('bluebird')
+const compareVersions = require('compare-versions')
 const EXIT_CODES = require('../shared/exit-codes')
 const buildWindowMenu = require('./menu')
 const settings = require('./app/models/settings')
 const analytics = require('./app/modules/analytics')
 const { getConfig } = require('../shared/utils')
+const { version, packageType } = require('../../package.json')
 /* eslint-disable lodash/prefer-lodash-method */
+/* eslint-disable no-magic-numbers */
 
 const config = settings.getDefaults()
-
 const configUrl = settings.get('configUrl') || 'https://balena.io/etcher/static/config.json'
+const packageTypes = [
+  'rpm',
+  'deb',
```

Comment:
Is this correct? I thought that when you install a deb (or rpm) package, it was then up to the system package-manager to keep it up to date, rather than having the package itself directly "auto update"?

Reply:
@lurch it is: those are the packages that don't support auto-update

Reply:
I should probably choose a better name, like `ignoredPackages` or something along those lines

Reply:
Does that mean that the `const packageUpdatable = _.includes(packageTypes, packageType)` logic is wrong?

Reply:
@lurch *big face palm*
It was previously a `indexOf ... < 0`, then I mixed things up.
Fixing
*end big face palm*